sdk/servicebus/service-bus/test/stress/serviceBusStressTester.ts

```diff
@@ -158,14 +161,16 @@ export class SBStressTestsBase {
         for (let i = 0; i < numberOfMessages; i++) {
           messages.push(generateMessage(useSessions, numberOfSessions));
         }
+
+        this.trackSentMessages(messages);
```

Comment:
This is effectively tracking messages that we'll attempt to send whether they succeed or not. Is that what's intended, or just successful sends?

Reply:
I'd like to rework this a bit, TBH. The issue is I was chasing is that the messages can be sent (async) and arrive (async) prior to the message tracking data being added. This is a somewhat odd little port.

It's not great this way either, but I hope to take another pass at it.